docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md

```diff
@@ -35,10 +35,6 @@ The majority of our feature flagging is blocks, this is a list of them:
 ### Experimental flag
 
 -   Product Gallery ([PHP flag](https://github.com/woocommerce/woocommerce-blocks/blob/e3fe996251b270d45ecc73207ea4ad587c2dbc78/src/BlockTypesController.php#L232) | [webpack flag](https://github.com/woocommerce/woocommerce-blocks/blob/e3fe996251b270d45ecc73207ea4ad587c2dbc78/bin/webpack-entries.js#L50-L52C3)).
--   Product Gallery Thumbnails ([PHP flag](https://github.com/woocommerce/woocommerce-blocks/blob/04af396b9aec5a915ad98188eded53e723a051d3/src/BlockTypesController.php#L234) | [webpack flag](https://github.com/woocommerce/woocommerce-blocks/blob/04af396b9aec5a915ad98188eded53e723a051d3/bin/webpack-entries.js#L57-L60)).
```

Comment:
@danieldudzic, the `Product Gallery Thumbnails` is still behind an experimental flag in trunk ([PHP](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/BlockTypesController.php#L236), [webpack](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/bin/webpack-entries.js#L66-L68)). Is this block supposed to be no longer experimental? Cc: @kmanijak